tutorials/Getting_Started/README.md

```diff
@@ -1,63 +1,61 @@
-# Getting Started with Compose for Desktop
+# Getting Started With Compose For Desktop
 
-## What is covered
+## What Is Covered
 
 In this tutorial we will create a simple desktop UI application
 using the Compose UI framework.
 
-## Prerequisites
+### Prerequisites
 
- Compose for Desktop can produce applications for macOS, Linux and Windows platforms,
-and so any of these platforms can be used for this tutorial.
+ Compose for Desktop can produce applications for macOS, Linux, and Windows platforms. So any of these platforms can be used for this tutorial.
 
 The following software has to be preinstalled:
    * JDK 11 or later
    * IntelliJ IDEA Community Edition or Ultimate Edition 20.2 or later (other editors could be used, but we assume you are using IntelliJ IDEA in this tutorial)
 
-## Creating a new project
+## Creating A New Project
 
-### New project wizard
+### New Project Wizard
 
-Kotlin support in IDEA IDE starting with the version 2020.3 comes with the new project wizard
-capable to create a Compose application automatically.
+Kotlin support in IDEA IDE starting with the version 2020.3, comes with the new project wizard which is
+capable of creating a Compose application automatically.
 
-Note that JDK must be at least JDK 11, and to use the native distribution
-packaging JDK 14 or later must be used.
+*Note that JDK must be at least JDK 11, and to use the native distribution
```

Comment:
* Please note that the JDK must be at least JDK 11, and to use the native distribution packaging JDK 14 or later must be used.*

Reply:
done